site/content/blogs/release-v126.en.md

```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
 
 ## Sidecar image build
 
-## Request-Driven Web Service secret support
+## Request-Driven Web Service secrets support
+You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
+```yaml
+  secrets:
+    secret-name-as-env-variable: secret-name-from-aws
+```
+And deploy! Your service can now access the secret as an environment variable.
+For more detailed use of the `secrets` field:
+```yaml
+secrets:
+  # SecretsManager
+  # (Recommended) Option 1. Referring to the secret by name.
+  DB:
+    secretsmanager: 'demo/test/mysql'
+  # You can refer to a specific key in the JSON blob.
+  DB_PASSWORD:
+    secretsmanager: 'demo/test/mysql:password::'
+  # You can substitute predefined environment variables to keep your manifest succinct.
+  DB_PASSWORD:
+        secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/mysql:password::'
+
+  # Option 2. Alternatively, you can refer to the secret by ARN.
+  DB: 'arn:aws:secretsmanager:us-west-2:111122223333:secret:demo/test/mysql-Yi6mvL'
+ 
+  #SSM Parameter Store
+  # (Recommended) Option 1. Referring to the secret by ARN.
```

Comment:
Why is this recommended? I would probably have recommended option 2 😂

Reply:
:P At App Runner side we mostly use ARNs. Should I remove the `Recommended` word ?

Reply:
Yes let's remove "Recommended" 👍🏼 !

Reply:
Done!